README.md

```diff
@@ -11,18 +11,18 @@ android {
 }
 ```
 
-Once you add the API Key, you can run the app. The app is ready to go for accessibility testing by tapping on the floating action button to kick off a scan at any point you'd like. <!-- Insert documentation link once new docs are updated -->.
+Once you add the API Key, you can run the app. The app is ready to go for accessibility testing by tapping on the floating action button to kick off a scan at any point you'd like. <!-- Insert documentation link once new docs are updated -->
```

Comment:
```suggestion
Once you add the API Key, you can run the app. The app is ready to go for accessibility testing by tapping on the floating action button to kick off a scan at any point you'd like.
```

Reply:
generally dunno what documentation I was trying to link to

Reply:
I went to look if we had anything!